--- libiconv-1.9.1/lib/relocatable.c.orig	2003-04-05 12:03:25.000000000 +0200
+++ libiconv-1.9.1/lib/relocatable.c	2003-12-01 21:56:01.319741592 +0100
@@ -38,6 +38,9 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <windef.h>
+#include <wtypes.h>
+#include <winbase.h>
 
 #ifdef NO_XMALLOC
 # define xmalloc malloc
--- libiconv-1.9.1/libcharset/lib/relocatable.c.orig	2003-04-05 12:03:25.000000000 +0200
+++ libiconv-1.9.1/libcharset/lib/relocatable.c	2003-12-01 21:56:24.326244072 +0100
@@ -38,6 +38,9 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <windef.h>
+#include <wtypes.h>
+#include <winbase.h>
 
 #ifdef NO_XMALLOC
 # define xmalloc malloc
